service/matching/task_queue_manager_test.go

```diff
@@ -272,27 +272,6 @@ func makeTestBlocAlloc(f func() (taskQueueState, error)) taskQueueManagerOpt {
 	return withIDBlockAllocator(&testIDBlockAlloc{alloc: f})
 }
 
-func TestSyncMatchLeasingUnavailable(t *testing.T) {
```

Comment:
The behavior tested by this test was removed a while ago, and the test still passed because the matcher wasn't stopped